sources/packages/web/src/store/modules/common/mutations.ts

```diff
@@ -0,0 +1,11 @@
+import { MutationTree } from "vuex";
+import { LoggedInUserClientTypeState, ClientIdType } from "@/types";
+
+export const mutations: MutationTree<LoggedInUserClientTypeState> = {
+  setClientTypeState(
+    state: LoggedInUserClientTypeState,
+    clientType: ClientIdType,
+  ) {
+    state.clientType = clientType;
```

Comment:
We already have the same globally available on _AuthService.shared.authClientType_.
This is also something that the user needs to do a new authentication to change, so it will not be changing.
Can you please elaborate on why we need add it to the store?